spec.html

```diff
@@ -30031,7 +30052,7 @@ <h1>String.prototype.indexOf ( _searchString_ [ , _position_ ] )</h1>
           1. Let _len_ be the length of _S_.
           1. Let _start_ be min(max(_pos_, 0), _len_).
           1. Let _searchLen_ be the length of _searchStr_.
-          1. Return the smallest possible integer _k_ not smaller than _start_ such that _k_ + _searchLen_ is not greater than _len_, and for all nonnegative integers _j_ less than _searchLen_, the code unit at index _k_ + _j_ within _S_ is the same as the code unit at index _j_ within _searchStr_; but if there is no such integer _k_, return the value -1.
+          1. Return ! StringIndexOf(_S_, _searchStr_, _start_).
```

Comment:
It would be nice to see similar use of StringIndexOf for String.prototype.includes. String.prototype.lastIndexOf would be the final piece of the full closure, but that's sufficiently different that I can see leaving it out.

Reply:
How about this?

```diff
diff --git a/spec.html b/spec.html
index c10bf37..c305fcd 100644
--- a/spec.html
+++ b/spec.html
@@ -30024,7 +30024,9 @@ THH:mm:ss.sss
           1. Let _len_ be the length of _S_.
           1. Let _start_ be min(max(_pos_, 0), _len_).
           1. Let _searchLen_ be the length of _searchStr_.
-          1. If there exists any integer _k_ not smaller than _start_ such that _k_ + _searchLen_ is not greater than _len_, and for all nonnegative integers _j_ less than _searchLen_, the code unit at index _k_ + _j_ within _S_ is the same as the code unit at index _j_ within _searchStr_, return *true*; but if there is no such integer _k_, return *false*.
+          1. Let _index_ be ! StringIndexOf(_S_, _searchStr_, _start_).
+          1. If _index_ &ge; 0, return *true*.
+          1. Return *false*.
         </emu-alg>
         <emu-note>
           <p>If _searchString_ appears as a substring of the result of converting this object to a String, at one or more indices that are greater than or equal to _position_, return *true*; otherwise, returns *false*. If _position_ is *undefined*, 0 is assumed, so as to search all of the String.</p>
```

Reply:
Looks great to me!

Reply:
Pushed up that change.